shared/src/commonMain/kotlin/co/touchlab/kampkit/metaweather/ktor/MetaWeatherApiImpl.kt

```diff
@@ -27,7 +27,6 @@ class MetaWeatherApiImpl : MetaWeatherApi {
                     level = LogLevel.ALL
                 }
             }
-            level = LogLevel.INFO
```

Comment:
Why remove this? This looks like a keeper for debugging purposes.

Reply:
it's being defined twice ^